### ISSUE TITLE
Possible improvement for ChainJobs

### ISSUE THREAD

**Issue by thijssnelleman** [FZJ-JSC/JUBE#17]
In the file https://github.com/FZJ-JSC/JUBE/blob/29aaf678069551dc6f26a775ee831852bef7d11f/platform/slurm/chainJobs.sh

The job ids are extracted after submitting to slurm. By running with --parseable, only the job ID will be returned, this could save some headache.

**Comment by filipesmg** [comment]
Indeed, here is an example (the argument is actually `--parsable`):
```
> sbatch --parsable submit.job
10663264
> sbatch submit.job
Submitted batch job 10663269
```
But I guess that will actually only change the last line in the script:
```
# the JOBID is the last field of the output line
echo ${JOBID##* } > $LOCKFILE
```
that could be written as
```
# the JOBID is the last field of the output line
echo $JOBID > $LOCKFILE
```
Is that correct? Is it different when a job has a dependency?

**Comment by thijssnelleman** [comment]
Exactly! I don't think there are any differences, sbatch just returns a job id after successful submission
